### PR TITLE
feat : image component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
Animation
**IMAGE COMPONENT**

- RESPONSIVE SQUARE IMAGE
- RESPONSIVE CIRCULAR IMAGE
- RESPONSIVE ANIMATED IMAGE

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -532,6 +532,53 @@ <h3 class="card-title">Our Chaning Work</h3>
 </code>
```
Comment (MaruthiPothuganti) [inline docs.html:533]: Use Prettier For Code formatting

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: Code Formatting, rest looks good

File: docs.html [hunk 2]
```diff
@@ -532,6 +532,53 @@ <h3 class="card-title">Our Chaning Work</h3>
...
 
 </section>
+
+<section class="section" >
+  <div class="component-title" >Image</div>
+<div class="component-detail" >
+  Responsive Image Components are used to dsiplay Images on websites, which are screen responsive. They expand upto the width of the container they are in.
```
Comment (junaidshaikh-js) [inline docs.html:539]: Look for spelling mistakes

**Review by Manpreet-hub — COMMENTED** [review]
looks good!

File: docs.html (hunk 1 above)
Comment (tanishk15sharma) [reply docs.html:533]: Yes I will use it now

File: style/components/image.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  90% {
+    transform: translate(1px, 2px) rotate(0deg);
+  }
+  100% {
+    transform: translate(1px, -2px) rotate(-1deg);
+  }
+}
+
+/* not a part of component , Just docx style */
```
Comment (junaidshaikh-js) [inline style/components/image.css:57]: Doc-related CSS should be in a separate folder.

**Review by junaidshaikh-js — COMMENTED** [review]
Use prettier for code formatiing

File: docs.html (hunk 2 above)
Comment (tanishk15sharma) [reply docs.html:539]: okya i will change this

**Pull request merged**